worlds/dark_souls_3/__init__.py

```diff
@@ -52,270 +45,431 @@ class DarkSouls3World(World):
     option_definitions = dark_souls_options
     topology_present: bool = True
     web = DarkSouls3Web()
-    data_version = 5
+    data_version = 6
     base_id = 100000
-    required_client_version = (0, 3, 7)
+    enabled_location_categories: Set[DS3LocationCategory] = set()
```

Comment:
```suggestion
    enabled_location_categories: Set[DS3LocationCategory]
```
it's better to not assign an empty set here to avoid accidentally using the global set for anything

Reply:
Fixed in 64aa83f